pkgs/development/libraries/qscintilla/default.nix

```
@@ -64,10 +64,10 @@ stdenv.mkDerivation rec {
       background colours and multiple fonts.
     '';
     homepage = "https://www.riverbankcomputing.com/software/qscintilla/intro";
-    license = with licenses; [ gpl3 ]; # and commercial
-    maintainers = with maintainers; [ peterhoeg ];
-    platforms = platforms.unix;
+    license = lib.licenses.gpl3;
+    maintainers = with lib.maintainers; [ peterhoeg ];
+    platforms = lib.platforms.linux;
```

```suggestion
    platforms = lib.platforms.unix
```

Darwin is supported by upstream.

---

Created https://github.com/NixOS/nixpkgs/pull/383983 to fix darwin build